app/src/main/java/io/intrepid/contest/screens/contestcreation/NewContestActivity.java

```diff
@@ -0,0 +1,160 @@
+
+import android.content.Context;
+import android.content.Intent;
+import android.os.Build;
+import android.support.annotation.NonNull;
+import android.support.annotation.RequiresApi;
+import android.support.annotation.StringRes;
+import android.support.v4.view.ViewPager;
+import android.support.v7.app.ActionBar;
+import android.support.v7.widget.Toolbar;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.Window;
+import android.view.WindowManager;
+
+import java.util.UUID;
+
+import butterknife.BindView;
+import io.intrepid.contest.R;
+import io.intrepid.contest.base.BaseMvpActivity;
+import io.intrepid.contest.base.PresenterConfiguration;
+import io.intrepid.contest.base.TextValidatableView;
+import io.intrepid.contest.models.Contest;
+import io.intrepid.contest.screens.contestcreation.addcategoriestocontest.AddCategoriesFragment;
+import io.intrepid.contest.screens.contestcreation.describecontest.DescribeContestFragment;
+import io.intrepid.contest.screens.contestcreation.namecontest.NameContestFragment;
+import io.intrepid.contest.screens.entrysubmission.entryimage.EntryImageActivity;
+import io.intrepid.contest.screens.splash.SplashActivity;
+import io.intrepid.contest.utils.SlidingTabAdapter;
+
+
+public class NewContestActivity extends BaseMvpActivity<NewContestPresenter> implements NewContestContract.View {
+
+    private static final String KEY_CREATOR_ID = "User_Id";
+    @BindView(R.id.toolbar)
+    Toolbar toolbar;
+    @BindView(R.id.fragment_container)
+    ViewPager viewPager;
+    private SlidingTabAdapter tabAdapter;
+
+    public static Intent createIntent(Context context, UUID creatorId) {
+        return new Intent(context, NewContestActivity.class).putExtra(KEY_CREATOR_ID, creatorId);
+    }
+
+    @NonNull
+    @Override
+    public NewContestPresenter createPresenter(PresenterConfiguration configuration) {
+        return new NewContestPresenter(this, configuration);
+    }
+
+    @Override
+    protected int getLayoutResourceId() {
+        return R.layout.activity_viewpager_container;
+    }
+
+
+    public NewContestContract.Presenter getPresenter() {
+        return presenter;
+    }
+
+    private void setupToolbar() {
+        setSupportActionBar(toolbar);
+        ActionBar actionBar = getSupportActionBar();
+        actionBar.setDisplayHomeAsUpEnabled(true);
+        actionBar.setTitle(R.string.new_contest);
+        actionBar.setHomeAsUpIndicator(android.R.drawable.ic_media_rew);
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+            setupStatusBarColor();
+        }
+    }
+
+    @RequiresApi(api = Build.VERSION_CODES.LOLLIPOP)
+    private void setupStatusBarColor() {
+        Window window = getWindow();
+        window.clearFlags(WindowManager.LayoutParams.FLAG_TRANSLUCENT_STATUS);
+        window.addFlags(WindowManager.LayoutParams.FLAG_DRAWS_SYSTEM_BAR_BACKGROUNDS);
+        window.setStatusBarColor(getResources().getColor(R.color.colorPrimaryDark));
```

Comment:
same Api 17 issues with used the non-deprecated method. Should we discussing moving up our min sdk? @gsintrepid @jmccormack200